recipes/ncbi-cxx-toolkit-public/26.0/test_package/conanfile.py

```diff
@@ -0,0 +1,20 @@
+import os
+
+from conans import ConanFile, CMake, tools
+
+
+class NcbiCxxToolkitTest(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake"
```

Comment:
```suggestion
    generators = "cmake", "cmake_find_package"
```

Let's use targets to find the libraries (and that way we also test that names are assigned properly)